src/RestaurantsPage.jsx

```diff
@@ -0,0 +1,32 @@
+import React, { useEffect } from 'react';
+
+import { useDispatch } from 'react-redux';
+
+import RegionsContainer from './RegionsContainer';
+import CategoriesContainer from './CategoriesContainer';
+import RestaurantsContainer from './RestaurantsContainer';
+
+import {
+  loadInitialData,
+} from './actions';
+
+// 0. 지역, 분류 목록을 얻기
+// 1. 지역 선택 - Regions <- API (0)
+// 2. 분류 선택 - Categories - 한식, 중식, 일식, ... <- API (0)
+// 3. 식당 목록 - Restaurants <- API (with region, category) -> 1, 2 모두 완료된 경우
+
+export default function RestaurantsPage() {
+  const dispatch = useDispatch();
+
+  useEffect(() => {
+    dispatch(loadInitialData([]));
+  });
```

Comment:
loadInitialData에 빈배열의 인자를 넘기라는 의미는 아니었습니다 😅

```suggestion
  useEffect(() => {
    dispatch(loadInitialData());
  }, []);
```

위와 같이 useEffect에 두번째 인자로 빈 배열을 넘길 경우, 최초 렌더 이후에 한번만 첫번째 인자로 넘기는 콜백을 실행하라는 의미입니다~ 문서를 한번 보시고 이해가 안되는 부분이 있다면 말씀주세요!

- https://ko.reactjs.org/docs/hooks-reference.html#useeffect

Reply:
ㅎㅎ.. 문서 읽어보겠습니다. 😅